projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/model/DataJobExecution.java

```diff
@@ -50,6 +50,8 @@ public class DataJobExecution {
 
   private String jobVersion;
 
+  private String jobPythonVersion;
+
```

Comment:
Should the be api.yaml changes?
VEP-1739 contains only Deployment API change, may include Execution API.

Reply:
It makes sense to me. I'll add a sentence for that in a separate PR.